license-classifications.yml

```diff
@@ -2195,11 +1701,6 @@ categorizations:
       - "property:include-in-notice-file"
       - "include-in-notice-file"
 
-  # https://spdx.org/licenses/Libtool-exception.html
-  - id: "Libtool-exception"
-    categories:
-      - "property:AutoConf-or-other-exception"
-
   - id: "LicenseRef-389-exception"
```

Comment:
Should this be deleted as well?

Reply:
Good question: check tool doesn't complain about it. @sschuberth ?

Reply:
It's a LicenseRef, so pretty much anything goes. This relates to our discussion on a script that checks the licenses against ScanCode's license database so we can delete licenses that are not included there.

Reply:
Our SPDX check in Clearance UI complains it's not a valid license.

Reply:
Yes, it should be deleted. The entry is "cheating" as it prefixes a stand-alone exception (which would not be allowed" with "LicenseRef-" so that "anything goes" as @mmurto said. However, it's easy to check for this case: For non-compound expressions, just strip any "LicenseRef-" prefix to check if the remained is a known exception.